### DISCUSSION TITLE
How to use spock versioning in dependabot

### DISCUSSION THREAD

**Discussion by huehnerlady** [spockframework/spock#1566] [Q&A]
Hi,

we are using dependabot to keep our dependencies up to date. 
Unfortunately so far we didn't have any luck working with spock updates.

We are using groovy 3 so we were on version `2.1-groovy-3.0`
`estImplementation("org.spockframework:spock-core:2.1-groovy-3.0")`

Dependabot however want to update to `2.3-groovy-4.0` instead of `2.3-groovy-3.0`
After using `@dependabot ignore this major version` we disabled dependabot for `2.x.x` instead of `x.x-groovy-4.0`

Are you using dependabot and maybe knows a trick on how to resolve our problem?

**Comment by szpak** [comment]
Personally, I use [Renovate](https://github.com/renovatebot/renovate/) which has mechanisms to handle that. See the [exact snippet](https://github.com/renovatebot/renovate/discussions/19747#discussioncomment-4697978) to solve the aforementioned problem (and also it [Renovate] nicely support grouping (while with dependabot it is somehow [more complicated](https://github.com/dependabot/dependabot-core/issues/1190)).
